### ISSUE TITLE
Export labels stats

### ISSUE THREAD

**Issue by omerlh** [summerwind/github-project-exporter#1]
It would be nice to see the count of issues exist per column per label. E.g. 10 issues with label "good first issue". What do you think?

**Comment by summerwind** [comment]
Good idea. But project card API that current code used does not contains the label information. So we need extra call to GitHub API to get the labels and need to care about API limit.

**Comment by omerlh** [comment]
Yes, that was my assumption - that we need to pull the actual card to get the metadata...
